docs/adminguide/frozen_users.rst

```diff
@@ -0,0 +1,114 @@
+.. Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 (eventually AGPL-3.0) 2016-present Scille SAS
+
+.. _doc_adminguide_frozen_users:
+
+
+Frozen users
+============
+
+It is possible for server administrators to freeze specific Parsec users.
+
+This feature is not part of the Parsec revocation system, but a way for system administrators to temporarily block users from connecting to the server while waiting for organization administrators to actually revoke them. It is exposed as an HTTP route that requires the administration token. This way, it is possible for existing directory services to automatically freeze users that have been removed from the directory.
+
+HTTP ``users`` route
+--------------------
+
+This route is made available as ``/administration/organizations/<raw_organization_id>/users`` and requires an administration token.
+
+It only supports the ``GET`` method which lists information for all users, including Parsec ID, user name, user email, and frozen status.
+
+Here's an example using ``curl`` and ``jq``:
+
+  .. code-block:: bash
+
+    $ curl https://$server/administration/organizations/$organization/users \
+        -H "Authorization: Bearer $administration_token" | jq
+
+A successful request returns a JSON dictionary similar to the one below:
+
+  .. code-block:: json
+
+    {
+      "users": [
+        {
+          "user_name": "Alice",
+          "frozen": true,
+          "user_email": "alice@example.com",
+          "user_id": "940a380aedd44127863d952a66cfce1e"
+        },
+        {
+          "user_name": "Bob",
+          "frozen": false,
+          "user_email": "bob@example.com",
+          "user_id": "7d04b0df51a74158b35b6386eecdf4e0"
+        }
+      ]
+    }
+
+HTTP ``users/frozen`` route
+---------------------------
+
+This route is made available as ``/administration/organizations/<raw_organization_id>/users/frozen`` and requires an administration token.
+
+It only supports the ``POST`` method which modifies the ``frozen`` status for a given user.
+
+Here's an example of generating the request data using ``jq``:
+
+  .. code-block:: bash
+
+    $ DATA=$(jq -n --arg user_id "$user_id" --argjson frozen true '$ARGS.named')
+    $ echo $DATA
+    {
+    "user_id": "940a380aedd44127863d952a66cfce1e",
+    "frozen": true
+    }
+
+The request can also use the ``user_email`` field instead of ``user_id`` to identify the Parsec user:
```

Comment:
We could add a link to the section (see comment below):
```suggestion
The request can also use the ``user_email`` field instead of ``user_id`` to identify the Parsec user (see :ref:`user-identification`):
```